src/__tests__/create-ssr-algolia-client.spec.ts

```diff
@@ -1,36 +1,51 @@
 import { createSSRSearchClient } from '../create-ssr-algolia-client';
 import * as algoliasearchProxy from 'algoliasearch/index';
-import { VERSION } from '../version';
-import { VERSION as AngularVersion } from '@angular/core';
 
 jest.mock('algoliasearch/index');
 
 describe('Create SSR', () => {
-  it('passes the User-Agent', () => {
```

Comment:
We should keep this test since it tests another behavior.

Reply:
oops, thank you :)